models/articles.models.js

```diff
@@ -17,18 +17,19 @@ exports.fetchArticleById = (article_id) => {
 		});
 };
 
-exports.fetchArticles = () => {
-	return db
-		.query(
-			`SELECT articles.* 
-			, COUNT(comments.comment_id) AS comment_count
-			FROM articles LEFT OUTER JOIN comments ON comments.article_id = articles.article_id
-			GROUP BY articles.article_id
-			ORDER BY created_at DESC`
-		)
-		.then((results) => {
-			return results.rows;
-		});
+exports.fetchArticles = (topic) => {
+	const queryValues = [];
+	let queryStr = `SELECT articles.*, COUNT(comments.comment_id) 
+	AS comment_count FROM articles LEFT OUTER JOIN comments 
+	ON comments.article_id = articles.article_id`;
```

Comment:
body should be ommitted here

Reply:
refactored for specified fields: 
`SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT...`